src/main/java/com/lambdaschool/wellness/controllers/CompetitionController.java

```diff
@@ -13,77 +13,61 @@
 import java.net.URI;
 import java.net.URISyntaxException;
 
-
+@RequestMapping(value = "/api/competition")
 @RestController
-@RequestMapping(value = "/api/groups")
-@CrossOrigin("http:localhost:3000")
-
 public class CompetitionController
 {
     @Autowired
     private CompetitionService competitionService;
 
     @GetMapping("/all")
-    public Iterable<Competition> getAllGroups()
+    public Iterable<Competition> getAllComp()
     {
         return competitionService.findAll();
     }
 
-    @GetMapping("/{groupid}")
-    public ResponseEntity<?> getGroupById(@PathVariable Long groupid)
+    @GetMapping("/{compid}")
+    public ResponseEntity<?> getByCompid(@PathVariable Long compid)
     {
-        Competition competition = competitionService.findById(groupid);
+        Competition competition = competitionService.findById(compid);
         return new ResponseEntity<Competition>(competition, HttpStatus.OK);
     }
 
+
     @PostMapping("")
-    public ResponseEntity<?> addNewGroup(@Valid @RequestBody Competition newGroup) throws URISyntaxException
+    public ResponseEntity<?> addNewComp(@Valid @RequestBody Competition newComp) throws URISyntaxException
     {
-        newGroup = competitionService.save(newGroup);
+        newComp = competitionService.save(newComp);
         HttpHeaders responseHeaders = new HttpHeaders();
-        URI newUserURI = ServletUriComponentsBuilder.fromCurrentRequest().path("/groupid").buildAndExpand(newGroup.getGroupid()).toUri();
+        URI newUserURI = ServletUriComponentsBuilder.fromCurrentRequest().path("/compid").buildAndExpand(newComp.getCompid()).toUri();
         responseHeaders.setLocation(newUserURI);
         return new ResponseEntity<>(responseHeaders, HttpStatus.CREATED);
```

Comment:
Same with groups. Would it be better to return the competition object in the response?